content/html/concepts/semantic-html/terms/main/main.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: '<main>'
+Description: 'Represents the primary content within the body element of the web page.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Semantic'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<main>` semantic tag represents the primary content within the `<body>` of the web page. This excludes content from other structural elements such as the [header](https://www.codecademy.com/resources/docs/html/semantic-html/header) or [footer](https://www.codecademy.com/resources/docs/html/semantic-html/footer).
+
```

Comment:
To stay consistent with the other semantic HTML terms, there should be a section for "Syntax"  here before giving the example.

Reply:
Good point, I'll add one! 😄